platforms/documentation/docs/src/docs/userguide/dep-man/advanced/variant_aware_resolution.adoc

```diff
@@ -366,6 +367,75 @@ Gradle knows the _consumer_ requires a Java 7-compatible library, but the _produ
 If the _consumer_ requested `org.gradle.jvm.version=15`, Gradle could choose either the Java 8 or Java 11 *variant*.
 Gradle would then select the highest compatible version—Java 11.
 
+[[sec:secondary-variants]]
+== Secondary Variants
+
+**Secondary variants** are *additional sets of artifacts* that a variant can expose, which do **not** participate in graph resolution.
+Instead, they are considered *only during artifact selection*.
+
+A secondary variants:
+
+* Is an additional artifact set exposed by a configuration.
+* It has its own attributes, artifacts, and optionally capabilities.
+* It must be explicitly published (secondary variants are not usually published or declared in metadata).
+* It may be selected via an `ArtifactView` or `ArtifactResolutionQuery`, if the requested attributes match.
```

Comment:
This is also not true. 

You _can_ use attributes on an artifact view to help pick artifact sets during artifact selection, but attributes on a configuration or on a dependency will contribute choosing which artifact set to select from a variant.